### ISSUE TITLE
[FEATURE]: Colour Checker Nano detection

### ISSUE THREAD

**Issue by TomaszSuchan** [colour-science/colour-checker-detection#74]
### Description

Would it be possible to add Color Checker nano detection? I tried tweaking the settings but I couldn't make it work. I also tried resizing the picture so the colour fields are square (they are rectangular in this Color Checkre version) but still no luck.

**Comment by KelSolaar** [comment]
Hi Tomasz,

We are working on a new detection process, slowly but steadily, and hopefully, it will work with it! Would it be possible to get some pictures to test with it please?

Cheers,

Thomas

**Comment by KelSolaar** [comment]
Hi Tomasz,

Getting there:

![image](https://github.com/colour-science/colour-checker-detection/assets/99779/7a4c153f-a9a8-4977-9e68-7a5ff61801bb)

Cheers,

Thomas

**Comment by TomaszSuchan** [comment]
Hi Thomas, thanks this looks promising!
I attach the picture with my use case. I just realised that the card I got has a bug, with the last colour in the second row wrong.
![P3151080](https://github.com/colour-science/colour-checker-detection/assets/9158028/202fdbd6-dc45-4a77-ad9d-202c3e760366)

**Comment by KelSolaar** [comment]
Ah yes, this is quite annoying as this is an important one (all of them are really in the 24 swatches checker)! Maybe you can exclude it for now when computing the transformation? I would check with X-Rite / Calibrite if it is possible to get an exchange.

**Comment by KelSolaar** [comment]
#75 was merged into `develop`, an invocation as follows should hopefully work:

```python
colour_checker_detection.detect_colour_checkers_segmentation(image, **colour_checker_detection.SETTINGS_SEGMENTATION_COLORCHECKER_NANO)
```

Let me know!

**Comment by KelSolaar** [comment]
This has been released in 0.2.0!
